### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by sylviamoss**
- Adds the badge to our discuss 
- Removes the unknown badge.
- Add logo
- Update example to HCL

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,25 @@
...
+[![Discuss](https://img.shields.io/badge/discuss-packer-3d89ff?style=flat)](https://discuss.hashicorp.com/c/packer)
 [![PkgGoDev](https://pkg.go.dev/badge/github.com/hashicorp/packer)](https://pkg.go.dev/github.com/hashicorp/packer)
 [![GoReportCard][report-badge]][report]
 [![codecov](https://codecov.io/gh/hashicorp/packer/branch/master/graph/badge.svg)](https://codecov.io/gh/hashicorp/packer)
 
 [circleci-badge]: https://circleci.com/gh/hashicorp/packer.svg?style=svg
 [circleci]: https://app.circleci.com/pipelines/github/hashicorp/packer
 [appveyor-badge]: https://ci.appveyor.com/api/projects/status/miavlgnp989e5obc/branch/master?svg=true
-[appveyor]: https://ci.appveyor.com/project/hashicorp/packer
+[appveyor]: https://ci.appveyor.com/project/hashicorp/packer
```
Comment (azr) [inline README.md:12]: Eh, actually, we don't build on appveyor anymore !

```suggestion
```

Comment (sylviamoss) [reply README.md:12]: I removed that one but forgot to remove from there! 
I will update more stuff on the readme like the JSON example and remove this one

Comment (azr) [reply README.md:12]: ( ah didn't see the wip ! Will let you be :D )

Comment (sylviamoss) [reply README.md:12]: I just added the WIP because I realized it has more things to update than that 😆

File: README.md [hunk 2]
```diff
@@ -1,23 +1,24 @@
...
 [report]: https://goreportcard.com/report/github.com/hashicorp/packer
 
-* Website: https://www.packer.io
-* IRC: `#packer-tool` on Freenode
-* Mailing list: [Google Groups](https://groups.google.com/forum/#!forum/packer-tool)
+<p align="center" style="text-align:center;">
+  <a href="https://www.packer.io">
+    <img alt="HashiCorp Packer logo" src="website/public/img/logo-hashicorp.svg" width="500" />
+  </a>
+</p>
```
Comment (azr) [inline README.md:21]: This logo almost disapears on dark mode:

Screenshot 2021-03-02 at 11 54 52

Should we do something about that ?

File: README.md [hunk 3]
```diff
@@ -47,33 +48,43 @@ yourself](https://github.com/hashicorp/packer/blob/master/.github/CONTRIBUTING.m
...
+  ami_name      = "packer-example ${local.timestamp}"
+  instance_type = "t2.micro"
+  region        = "us-east-1"
+  secret_key    = "${var.secret_key}"
+  source_ami    = "ami-af22d9b9"
+  ssh_username  = "ubuntu"
+}
+
+build {
+  sources = ["source.amazon-ebs.quick-start"]
```
Comment (azr) [inline README.md:80]: 👍🏼 nice !

**Review by azr — APPROVED** [review]
Just had a visual question about the Packer logo, otherwise LGTM ! Nice one !

File: README.md (hunk 2 above)
Comment (sylviamoss) [reply README.md:21]: 🤦🏼  oh no! We don't have the white version of this same logo. Right now there's no way to fix it.

Comment (sylviamoss) [reply README.md:21]: It looks so good on light mode 😭

Comment (sylviamoss) [reply README.md:21]: https://github.com/drcmda/dark-mode-github-readme-logos

**Pull request merged**